components/_Progress-Bar/progress-bar-initialization.js

```diff
@@ -0,0 +1,111 @@
+    if(lightThemeProgressBarEl) {
+        const lightThemeProgressBar = Object.create(window.msdsProgressBar);
+
+        let lightThemeBaseNumberElement = document.querySelectorAll('#LightThemeBaseNumber')[0]    
+        lightThemeBaseNumberElement.value = 12;
+
+        lightThemeProgressBar.init("LightThemeProgressBar", lightThemeBaseNumberElement.value, 0);  
+        
+        document.querySelector('#LightThemePrevStep').addEventListener('click', event => {            
+            lightThemeProgressBar.prev();
+        })
+    
+        document.querySelector('#LightThemeNextStep').addEventListener('click', event => {
+            lightThemeProgressBar.next();
+        })
+    
+        document.querySelector('#LightThemeBaseNumber').addEventListener('change', event => {
+            lightThemeProgressBar.init("LightThemeProgressBar", event.target.value, 0);        
+        })    
+    
+        const lightThemeStartFromOneElement = document.querySelector('#LightThemeStartFromOne');
+        lightThemeStartFromOneElement.addEventListener('change', event => {        
+            lightThemeProgressBar.startFromOne = lightThemeStartFromOneElement.checked;
+            const baseNumber = document.querySelectorAll('#LightThemeBaseNumber')[0].value;
+            lightThemeProgressBar.init("LightThemeProgressBar", baseNumber, 0);
+        })
+    }               
+          
+    
+    //Dark Theme scripts------------------        
+    //Initialize the progress bar with 12 steps.
+    const darkThemeProgressBarEl = document.getElementById("DarkThemeProgressBar");
+    if(darkThemeProgressBarEl) {
+        const darkThemeProgressBar = Object.create(window.msdsProgressBar);
+
+        let darkThemeBaseNumberElement = document.querySelectorAll('#DarkThemeBaseNumber')[0]    
+        darkThemeBaseNumberElement.value = 12;
+        
+        darkThemeProgressBar.init("DarkThemeProgressBar", darkThemeBaseNumberElement.value, 0); 
+
+        document.querySelector('#DarkThemePrevStep').addEventListener('click', event => {
+            darkThemeProgressBar.prev();
+        })
+    
+        document.querySelector('#DarkThemeNextStep').addEventListener('click', event => {
+            darkThemeProgressBar.next();
+        })
+    
+        document.querySelector('#DarkThemeBaseNumber').addEventListener('change', event => {
+            darkThemeProgressBar.init("DarkThemeProgressBar", event.target.value, 0);        
+        })    
+    
+        const darkThemeStartFromOneElement = document.querySelector('#DarkThemeStartFromOne');
+        darkThemeStartFromOneElement.addEventListener('change', event => {        
+            darkThemeProgressBar.startFromOne = darkThemeStartFromOneElement.checked;
+            const baseNumber = document.querySelectorAll('#DarkThemeBaseNumber')[0].value;
+            darkThemeProgressBar.init("DarkThemeProgressBar", baseNumber, 0);
+        })
+    }
+});  
```

Comment:
We should break this up into a couple of JS files, 
progress-bar-light-theme.js
progress-bar-dark-theme.js
etc.

So we know exactly which js belongs to each .hbs file